### PR TITLE
Add ownerGroupId to mysql recordset table

### PR DESCRIPTION
**Pull request by varsharun**
Fixes #426 .

Changes in this pull request:
- Added a new alter query under migrations section
- Updated MySqlRecordSetRepository to include the optional new field `owner-group-id` 
- Integration tests

### CODE REVIEW

File: modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala [hunk 1]
```diff
@@ -92,6 +92,7 @@ class MySqlRecordSetRepository extends RecordSetRepository with Monitored {
             fromRecordType(oldRs.typ),
             toPB(oldRs).toByteArray,
             toFQDN(change.zone.name, oldRs.name),
+            oldRs.ownerGroupId,
```
Comment (pauljamescleary) [inline modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: can you add a test for when ownerGroupId is `None`

I presume that `None` maps to `NULL` but best to make sure.

Comment (rebstar6) [reply modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: when we get a recordset, we are pulling from the blob anyway. until we are querying the index i dont know that we can test that

Comment (rebstar6) [reply modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: may be worth just querying the local mysql to validate this though by hand

Comment (pauljamescleary) [reply modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: I meant inserting / UPDATE a null value doesn't blow up the insert and update statement itself.

Here, the updated SQL statement is inserting the owner group id as a column.  Want to be sure that inserting None works without a failure

Comment (rebstar6) [reply modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: fyi, ran locally and this is what that value looks like:

```
mysql> select name, owner_group_id from vinyldns.recordset;
+----------------+--------------------------------------+
| name           | owner_group_id                       |
+----------------+--------------------------------------+
| foo            | NULL                                 |
| jenkins        | NULL                                 |
| ok.            | NULL                                 |
| already-exists | NULL                                 |
| asdasd         | 06424926-769f-4eee-8b80-adfd54108f32 |
| ok.            | NULL                                 |
| ok.            | NULL                                 |
| test           | NULL                                 |
+----------------+--------------------------------------+
8 rows in set (0.00 sec)

```

i think were good :)

Comment (rebstar6) [reply modules/mysql/src/main/scala/vinyldns/mysql/repository/MySqlRecordSetRepository.scala]: also of note:
```
mysql> SHOW INDEX FROM vinyldns.recordset
    -> ;
+-----------+------------+----------------------+--------------+----------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| Table     | Non_unique | Key_name             | Seq_in_index | Column_name    | Collation | Cardinality | Sub_part | Packed | Null | Index_type | Comment | Index_comment |
+-----------+------------+----------------------+--------------+----------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| recordset |          0 | PRIMARY              |            1 | id             | A         |          16 |     NULL | NULL   |      | BTREE      |         |               |
| recordset |          1 | zone_id_name_index   |            1 | zone_id        | A         |           2 |     NULL | NULL   |      | BTREE      |         |               |
| recordset |          1 | zone_id_name_index   |            2 | name           | A         |          11 |     NULL | NULL   |      | BTREE      |         |               |
| recordset |          1 | zone_id_name_index   |            3 | type           | A         |          16 |     NULL | NULL   |      | BTREE      |         |               |
| recordset |          1 | fqdn_index           |            1 | fqdn           | A         |          11 |     NULL | NULL   |      | BTREE      |         |               |
| recordset |          1 | owner_group_id_index |            1 | owner_group_id | A         |           2 |     NULL | NULL   | YES  | BTREE      |         |               |
+-----------+------------+----------------------+--------------+----------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
6 rows in set (0.00 sec)

```

File: modules/mysql/src/it/scala/vinyldns/mysql/repository/MySqlRecordSetRepositoryIntegrationSpec.scala [hunk 2]
```diff
@@ -272,6 +272,50 @@ class MySqlRecordSetRepositoryIntegrationSpec
       repo.getRecordSet(okZone.id, inserts(1).recordSet.id).unsafeRunSync().map(_.name) shouldBe
         Some(inserts(1).recordSet.name)
     }
+    "works when inserting owner-group-id" in {
```
Comment (mitruly) [inline modules/mysql/src/it/scala/vinyldns/mysql/repository/MySqlRecordSetRepositoryIntegrationSpec.scala]: Nitpick: can we change "owner-group-id" to "ownerGroupId" or "owner group ID"?
